src/components/header.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+
+import LoginButton from "./login-button";
+import LogoutButton from "./logout-button";
+
+export default async function Header() {
+  return (
```

Comment:
To tie it all together, you can do something like the following to render the relevant buttons after fetching the session:

```javascript
import Image from "next/image";

import LoginButton from "~/components/login-button";
import LogoutButton from "~/components/logout-button";
import { getServerSession } from "next-auth";

export default async function Header() {
  const session = await getServerSession();

  return (
    <header className="bg-cooper-blue-400 flex h-16 w-full items-center justify-between border border-blue-500 p-4">
      <h1 className="text-3xl font-extrabold text-white">COOPER</h1>
      {session ? <LogoutButton /> : <LoginButton />}
      {/* ... */}
   );
}
```